csharp/src/Ice/IObjectPrx.cs

```diff
@@ -494,4 +525,308 @@ protected ObjectPrx(SerializationInfo info, StreamingContext context)
             IceReference = communicator.CreateReference(str, null);
         }
     }
+
```

Comment:
I think we should move all this code to a separate file, `OutgoingRequest.cs`.

Reply:
fixed